src/dcqc/suites/suites.py

```diff
@@ -28,3 +28,8 @@ class TiffSuite(FileSuite):
 class OmeTiffSuite(TiffSuite):
     file_type = FileType.get_file_type("OME-TIFF")
     add_tests = (tests.OmeXmlSchemaTest, tests.BioFormatsInfoTest)
+
+class RiskTermsSuite(FileSuite):
```

Comment:
Thanks @adamjtaylor for the contribution!  We haven't officially discussed internally how we would support non-filetype suites.  I do potentially see this to be useful. Just so we can better assist, what was your rationale behind creating this class?

Reply:
This raises an important point. So far, we've focused on file formats like JSON, TIFF, and OME-TIFF. However, there are tests that are less related to a specific format and more related to a "data type". For example, confirming the strandedness of a pair of RNA-seq FASTQ files combines the FASTQ file format with the RNA-seq data type. 

Similarly, this PR introduces (I think) the concept of a plain text file that might contain sensitive information, such as dates or times. Let's imagine a `PhiFree` data type. Thinking out loud here, this kind of check might be adapted based on the file format. For example, in the case of CSV/TSV files, maybe you restrict the test to the headers, whereas you would only check keys in the case of a JSON file. 

All this to say that `dcqc` should be able to handle both concepts (file formats and data types) as well as their interaction (_e.g._ inferring the strandedness for an RNA-seq experiment is different depending on whether you have FASTQ or BAM/CRAM files).

Reply:
Thanks @thomasyu888 and @BrunoGrandePhD!

My rationale for the `RiskTermSuite` class was that it is conceivable that there are other risky strings that we might want to generically detect within any file using grep.

To @BrunoGrandePhD's point I was thinking that tasks within RiskTermSuite might need to be applied to several different file types, and possibly all file types. I concur with the concept of applying  test in different ways depending on the file type.

If easier I would be happy to restrict the suite/test to OME-TIFF or TIFF files. I will also need it to support other imaging formats in the near future including SCN and SVS.

Reply:
Thanks @adamjtaylor , this makes sense.   My 2 cents is to add the tests to OME-TIFF and TIFF files for now, which will give us time to think more deeply about how we want to support data type suites.

There is also another use case which is "project specific suites".